Formula/infer.rb

```diff
@@ -37,7 +50,7 @@ def install
     # builds in its own parallelization logic to mitigate that.
     ENV.deparallelize
 
-    ENV["INFER_CONFIGURE_OPTS"] = "--prefix=#{prefix} --disable-ocaml-binannot"
+    ENV["INFER_CONFIGURE_OPTS"] = "--prefix=#{prefix} --disable-ocaml-binannot --without-fcp-clang"
```

Comment:
The extra argument is probably not needed since we're not using the release tarball, right? But I don't think it will hurt either.

Reply:
or is it still needed due to the `    touch ".release"`?

Reply:
I'll add a comment, yes it's needed because of `touch ".release"`, which by default makes the build attempt to use the pre-built clang.